dataeng/resources/remote-config.sh

```diff
@@ -92,12 +92,31 @@ unassume_role
 #     }
 # }
 
+# set up token path
+export VAULT_TOKEN_PATH=${WORKSPACE}/vault-config/vault-token
+# set path for config file which store token in each job workspace
+export VAULT_CONFIG_PATH=${WORKSPACE}/vault-config/vault_config
```

Comment:
This config file is where the `token_helper` config is defined? How is the config file generated? I am not seeing it tracked in the git history.

Reply:
This is generate from vault-config.sh file

Reply:
https://github.com/edx/jenkins-job-dsl/blob/master/dataeng/resources/vault-config.sh